examples/luigi.cfg

```diff
@@ -49,13 +49,13 @@ with_realistic_diameters = False
 
 [Annotate]
 tag_output = False
-dataset_df = out_curated/curated_dataset.csv
+# dataset_df = out_curated/curated_dataset.csv  # This can be used to annotated the results of a previously run workflow
 result_path = out_annotated
 report_path = report_annotation.pdf
 nb_processes = 4
 
 [MType]
-mtype_regex = ["L[1-6]_[A-Z]+$", "L[1-6]_[A-Z]+:[A-C]$", "L23_[A-Z]+$", "L23_[A-Z]+:[A-C]$", "L[1-6]_[A-Z]+-[A-Z]+$"]
+; mtype_regex = ["L[1-6]_[A-Z]+$", "L[1-6]_[A-Z]+:[A-C]$", "L23_[A-Z]+$", "L23_[A-Z]+:[A-C]$", "L[1-6]_[A-Z]+-[A-Z]+$"]
```

Comment:
why not #? is this also a comment?

Reply:
Ah yes, my editor uses `;` by default in cfg files but I also prefer `#`.
Anyway, I forgot to uncomment this line, it was just to test the missing mtype case.